qp_woltka/woltka.py

```diff
@@ -219,19 +249,34 @@ def woltka_to_array(files, output, database_bowtie2, prep, url, name):
     # https://github.com/BenLangmead/bowtie2/issues/311
     preparation_information = join(output, 'prep_info.tsv')
     prep.set_index('sample_name').to_csv(preparation_information, sep='\t')
-    bowtie2 = (
-        f'mxdx mux --file-map {files_list_fp} --batch '
-        '${SLURM_ARRAY_TASK_ID} '
-        f'--batch-size {BATCHSIZE} --paired-handling interleave | '
-        'bowtie2 -p ${bt2_cores} '
-        f'-x {database_bowtie2} --interleaved - --seed 42 '
-        '--very-sensitive -k 16 --np 1 --mp "1,1" --rdg "0,1" --rfg "0,1" '
-        '--score-min "L,0,-0.05" --no-head --no-unal --no-exact-upfront '
-        "--no-1mm-upfront | cut -f1-9 | sed 's/$/\t*\t*/' | "
-        f'mxdx demux --file-map {files_list_fp} '
-        '--batch ${SLURM_ARRAY_TASK_ID} '
-        f'--batch-size {BATCHSIZE} --output-base {output}/alignments '
-        '--extension sam.xz')
+    if rev_exists:
```

Comment:
The two bowtie2 lines seem very similar - is it possible to isolate the differences in them more?

Reply:
I personally prefer to have it like this as it's more clear to me ... basically having the full command there. However, if you feel strongly about this change, I can change it.

Reply:
That's okay. I think if it's easier for you to read it this way it's better to keep it this way. Ty!